eng/common/TestResources/New-TestResources.ps1

```diff
@@ -93,6 +93,11 @@ param (
     })]
     [array] $AllowIpRanges = @(),
 
+    # Instead of running the post script, create a wrapped file to run it with parameters
+    # so that CI can run it in a subsequent step with a refreshed azure login
+    [Parameter()]
+    [switch] $SelfContainedPostScript,
```

Comment:
I suspect we need a better name for this as it is I'm not sure I could figure it out by name. Perhaps something that indicates you are writing the post script to a wrapper and let this be the name of that wrappers script so we also don't have to have the magic variable writing here either as it will be an input and the caller can control it.

Reply:
Ah ok, that's a simpler approach. So it's only hardcoded in the deploy yaml basically, and not passed around as a var.

Reply:
Actually this won't work, because we still need to call the script by it's path, and the path is not just `service directory + name` but rather `service directory + template file directory + name` which is calculated in the test resources script (https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/TestResources/New-TestResources.ps1#L627).

I'm not sure if there are any places that this would cause trouble with at the moment, but I figured that the implementation should honor template file path.

Reply:
Why wouldn't we hard-code the full path in the generated script?